src/promptflow-evals/promptflow/evals/_constants.py

```diff
@@ -23,4 +23,5 @@ class Prefixes:
 
 CONTENT_SAFETY_DEFECT_RATE_THRESHOLD_DEFAULT = 4
 
-BATCH_RUN_TIMEOUT = 3600
+BATCH_RUN_TIMEOUT_DEFAULT = 3600
+BATCH_RUN_TIMEOUT_ENV_VAR = "BATCH_RUN_TIMEOUT"
```

Comment:
May prefix with 'PF', like: 
PF_BATCH_RUN_TIMEOUT

Reply:
Actually there seems to already have an env called this.
PF_BATCH_TIMEOUT_SEC

Reply:
updated